### PR TITLE
Add trust proxy config support

### PR DESCRIPTION
**Pull request by djMax**

### CODE REVIEW

File: config/production.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "gracefulShutdownTimeout": 20,
+  "trustProxy": true,
```
Comment (jasisk) [inline config/production.json]: What about:
```
"trustProxy": ["172.32.0.0/16", "172.30.0.0/16"]
```

Comment (djMax) [reply config/production.json]: This is the open source version. We can make that setting in gb-services

Comment (djMax) [reply config/production.json]: Figure most people are behind a proxy, so this seems a reasonable default? Or we could have no default in @gasbuddy/service but support checking it

Comment (jasisk) [reply config/production.json]: > This is the open source version. We can make that setting in gb-services

Totally. My mistake. Deleting the evidence.

Comment (jasisk) [reply config/production.json]: > Figure most people are behind a proxy, so this seems a reasonable default? Or we could have no default in but support checking it

Eh. It's a little dangerous to make as a default. I'd default off but document.

**Pull request merged**